unit-tests/test-fw-update.py

```diff
@@ -88,24 +111,21 @@ def reset_update_counter( device ):
 device = devices.get_first( sn_list ).handle
 log.d( 'found:', device )
 product_line = device.get_info( rs.camera_info.product_line )
+product_name = device.get_info( rs.camera_info.name )
 log.d( 'product line:', product_line )
-
+log.d( 'product name:', product_name )
```

Comment:
Not needed -- you have a line in the output:
```
-D- found: <pyrealsense2.device: Intel RealSense D435 (S/N: 938422071384  FW: 05.12.14.50  UNLOCKED  on USB3.2)>
```

Reply:
done